### DISCUSSION TITLE
Form validation error alternative

### DISCUSSION THREAD

**Discussion by mpgr04** [fomantic/Fomantic-UI#541] [Behaviors (Api, Form Validation, Visibility)] [answered]
# Help Wanted

## Problem
Is there an alternative for using the ui error message div inside the form to display errors?
Because if i have a form with many fields, this div pushes generates a scrollbar. Is there some sort to show an alert or something with the errors which get triggered by the validation?

**Comment by ColinFrick** [comment, marked as answer]
@mpgr04 You could remove the error div and use onFailure to display a toast message: https://jsfiddle.net/5nr98sue/

**Comment by mpgr04** [comment]
Learned something new again!
Thank you very much!
